execution/execution.go

```diff
@@ -196,10 +196,14 @@ func newSubqueryFunction(e *parser.Call, t *parser.SubqueryExpr, storage storage
 	if parse.IsExtFunction(e.Func.Name) {
 		return nil, parse.ErrNotImplemented
 	}
-	// TODO: We dont pass arguments yet
 	if e.Func.Name == "quantile_over_time" {
-		return nil, parse.ErrNotImplemented
+		err := errors.Wrapf(parse.ErrNotSupportedExpr, "quantile_over_time with expression as first argument is not supported")
+		_, unwrapErr := extscanners.UnwrapConstVal(e.Args[0])
+		if unwrapErr != nil {
+			return nil, err
```

Comment:
can we just return err here without defining it up there?

Reply:
Yep, totally. Inlined it.